config/types.go

```diff
@@ -0,0 +1,126 @@
+// Copyright 2016 CoreOS, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package config
+
+type Config struct {
+	Ignition struct {
+		Config struct {
+			Append  []ConfigReference `yaml:"append"`
+			Replace *ConfigReference  `yaml:"replace"`
+		} `yaml:"config"`
+	} `yaml:"ignition"`
+	Storage struct {
+		Disks []struct {
+			Device     string `yaml:"device"`
+			WipeTable  bool   `yaml:"wipe_table"`
+			Partitions []struct {
+				Label    string `yaml:"label"`
+				Number   int    `yaml:"number"`
+				Size     string `yaml:"size"`
```

Comment:
Since this is a string, the plan is to continue supporting MiB, GiB dimensions I'd assume